### DISCUSSION TITLE
Saved map not showing up anywhere

### DISCUSSION THREAD

**Discussion by KendraFrancis** [jgraph/drawio-desktop#1356] [General]
Hi everyone,

Today I saved a mind map I was working on to my device, and somehow it got deleted...like from everything. I kept the tab open, didn't touch it for a few hours, and now that I need to open it, it doesn't pull up anywhere. 
- Saved to device
- Doesn't open in browser history
- - - When I tried to open through browser history it had the correct file name, so I clicked it. It popped up with an "Open New" window instead of the project, and when I went to go retry this step the file name in browser history was draw.io instead of the file name. 
- Opens through file saved on device, but it doesn't have content. It might as well be a new document. 

Can anyone help???

**Comment by davidjgraph** [comment]
You saved to your local computer while the content was still visible? Can you attach the zipped file of that save?

**Comment by KendraFrancis** [comment]
Sure, is this a common issue? And does saving it differently affect whether
it will open or not in the app?

**Comment by KendraFrancis** [comment]
Hi David, I figured out the issue:

The red text bar at the top of the app that reminds users to save does not
work. When hovered on, it reads "click here to save," but does not save.
The screen shot won't register the hover text, but even when immediately
clicked on it still reads "last saved _(several)_mins ago."
I clicked on this button several times yesterday, so it makes sense that
none of it was saved.
I believe this is a bug.
However, on a positive note, when saved manually under File - Save/Save As
it did save correctly.

Hope this helps, thanks for your reply!
